otterdog/eclipse-thingweb.jsonnet

```diff
@@ -17,6 +17,20 @@ orgs.newOrg('eclipse-thingweb') {
     web_commit_signoff_required: false,
   },
   _repositories+:: [
+    orgs.newRepo('.github') {
+      allow_update_branch: false,
+      default_branch: "main",
+      dependabot_security_updates_enabled: true,
```

Comment:
In my understanding, this is a no-code repository, right? Do we need dependabot?

Reply:
Yup only readme at best. Removing